index.html

```diff
@@ -11,26 +11,141 @@
+                or sign-ups required.</p>
+            <ul class="languages">
+                <li class="languages">html</li>
+                <li class="languages">css</li>
+                <li class="languages">javascript</li>
+            </ul>
+            <p class="worksee">See Project</p>
+        </article>
+
+        <article class="work">
+            <header class="work">
+                <img class="work" src="Images/Snapshoot-Portfolio2.png" alt="Avaiability"/>
+                <h1 class="work">Multi-Post Stories</h1>
+            </header>
+            <ul class="worklist">
+                <li class="card-title">CANOPY</li>
+                <li class="counter"><img src="Images/Counter.png" alt="Counter"/></li>
+                <li class="work">Back End Dev</li>
+                <li class="counter"><img src="Images/Counter.png" alt="Counter"/></li>
+                <li class="work">2015</li>
+            </ul>
+            <p class="work">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages">
+                <li class="languages">html</li>
+                <li class="languages">css</li>
+                <li class="languages">javascript</li>
+            </ul>
+            <p class="worksee" >See Project</p>
+        </article>
+
+    <article class="work">
+        <header class="work">
+            <img class="work" src="Images/Snapshoot-Portfolio3.png" alt="GymFit"/>
+            <h1 class="work">Tonic</h1>
+        </header>
+        <ul class="worklist">
+            <li class="card-title">CANOPY</li>
+            <li class="counter"><img src="Images/Counter.png" alt="Counter"/></li>
+            <li class="work">Back End Dev</li>
+            <li class="counter"><img src="Images/Counter.png" alt="Counter"/></li>
+            <li class="work">2015</li>
+        </ul>
+        <p class="work">A daily selection of privately personalized reads; no accounts 
+            or sign-ups required.</p>
+        <ul class="languages">
+            <li class="languages">html</li>
+            <li class="languages">css</li>
+            <li class="languages">javascript</li>
+        </ul>
+        <p class="worksee" >See Project</p>
+    </article>
+
+    <article class="work">
+        <header class="work">
+            <img class="work" src="Images/Snapshoot-Portfolio4.png" alt="Professional Art Printing"/>
+            <h1 class="work">Multi-Post Stories</h1>
+        </header>
+        <ul class="worklist">
+            <li class="card-title">CANOPY</li>
+            <li class="counter"><img src="Images/Counter.png" alt="Counter"/></li>
+            <li class="work">Back End Dev</li>
+            <li class="counter"><img src="Images/Counter.png" alt="Counter"/></li>
+            <li class="work">2015</li>
+        </ul>
+        <p class="work">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="languages">
```

Comment:
- [ ] kindly I would recommend you to stick to Figma design as much as possible to match it so that your project looks professional .you can do so by using the same margins, font colors, font style, padding, etc.